docs/query_engine/config/annotated_config.yaml

```diff
@@ -0,0 +1,84 @@
+listenAddress:
+  type: "config"
+  value: "0.0.0.0:7201"
+
+# metrics configuration
+metrics:
+  scope:
+    prefix: "coordinator"
+  prometheus:
+    handlerPath: /metrics
+    listenAddress: 0.0.0.0:7203 # until https://github.com/m3db/m3/issues/682 is resolved
+  sanitization: prometheus
+  samplingRate: 1.0
+  extended: none
+
+clusters:
```

Comment:
Bunch of these aren't annotated!

Reply:
I'm not doing everything! But yeah, I wanted to get this in so that @andrewmains12 can add his stuff.